nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Updater/SdkPackageUpdater.cs

```diff
@@ -222,17 +222,21 @@ private static void PinTransitiveDependency(ProjectBuildFile directoryPackages,
         directoryPackages.Update(updatedXml);
     }
 
-    private static async Task AddTransitiveDependencyAsync(string projectPath, string dependencyName, string newDependencyVersion, Logger logger)
+    private static async Task AddTransitiveDependencyAsync(string repoRootPath, string projectPath, string dependencyName, string newDependencyVersion, Logger logger)
```

Comment:
This function diff looks nasty, but it just wraps the existing behavior in the new function that sidelines all `global.json` files.